packages/modules/vehicles/leaf/soc.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+from typing import List
+
+import logging
+import time
+
+from helpermodules.cli import run_using_positional_cli_args
+from modules.common import store
+from modules.common.abstract_device import DeviceDescriptor
+from modules.common.abstract_vehicle import VehicleUpdateData
+from modules.common.component_state import CarState
+from modules.common.configurable_vehicle import ConfigurableVehicle
+from modules.vehicles.leaf.config import LeafSoc, LeafConfiguration
+
+from modules.vehicles.leaf import pycarwings2
```

Comment:
```suggestion
import pycarwings2
```
Und den Import auf das installierte Paket ändern.